packages/ra-core/src/controller/field/ReferenceArrayFieldController.tsx

```diff
@@ -24,7 +25,9 @@ interface Props {
  */
 const ReferenceArrayFieldController: FunctionComponent<Props> = props => {
     const { children, ...rest } = props;
+    const { resource } = useResourceContext(props);
     const controllerProps = useReferenceArrayFieldController({
+        resource,
```

Comment:
why doesn't the useReferenceArrayFieldController call the useResourceContext itself?